packages/build-tools/src/steps/functions/sendSlackMessage.ts

```diff
@@ -22,6 +27,23 @@ export function createSendSlackMessageFunction(): BuildFunction {
     fn: async (stepCtx, { inputs, env }) => {
       const { logger } = stepCtx;
       const slackMessage = inputs.message.value as string;
+      const slackPayload = inputs.payload.value as object;
+      if (!slackMessage && !slackPayload) {
+        logger.warn(
+          'You need to provide either "message" input or "payload" input to specify the Slack message contents'
+        );
+        throw new Error(
+          'You need to provide either "message" input or "payload" input to specify the Slack message contents'
+        );
```

Comment:
Is it necessary to have both of them?

Reply:
Probably not, since the error is logged anyway